### ISSUE TITLE
Lack of Logo for Mail

### ISSUE THREAD

**Issue by Minimalistic** [mikel/mail#1081]
Hello, I was wondering if you would be interested in a logo/brand image for the ruby mail library.  I haven't been contributing to repositories/projects for very long and am unsure how one would normally go about proposing such an addition.  Is doing it via creating a new issue an appropriate method?  
![ruby_mail_logo](https://cloud.githubusercontent.com/assets/3077684/23734592/9b5e8742-0445-11e7-9ced-efec3ad506bd.png)
Thoughts?  Thanks!

**Comment by mikel** [comment]
Hey there, nice :)

Are you proposing to donate this to the project?  If so, please update the ticket and let me know.

**Comment by Minimalistic** [comment]
Yep, I would.  I'm still fairly new to Github, when you say update the ticket - is that just making a comment in this issue thread?

**Comment by Minimalistic** [comment]
@mikel Not sure if this is still of interest to you, but if you ever end up wanting the logo just let me know how you want me to deliver it to you, your project is more than welcome to it.
